### DISCUSSION TITLE
run after compiled

### DISCUSSION THREAD

**Discussion by drdedou** [seleniumbase/SeleniumBase#2588] [Q&A] [answered]
I use disable_csp=True 
when I start browser Chrome I have this issue :
![Screenshot 2024-03-10 170711](https://github.com/seleniumbase/SeleniumBase/assets/47856801/7c82e8c8-afba-4d49-9c61-687d5acfe28e)

when I start edge browser I have this issue : 
![Screenshot 2024-03-10 170833](https://github.com/seleniumbase/SeleniumBase/assets/47856801/2a0f9477-d0b4-41eb-80a3-372e11951407)

my setting : 
    with SB(uc=True,undetected=True,browser=typeOfBrowser,start_page=GetUrl(),maximize=True,agent=UserAgent,locale_code=LocaleCode ,mobile=True,disable_csp=True) as sb:

The program works fine in debug mode

**Comment by mdmintz** [comment, marked as answer]
If you see "Chrome is being controlled by automated test software.", then it means you passed invalid arguments into initialization, and the browser launcher is defaulting to using regular Selenium with no special options or modes added. Since I can't see what you used for all your variables, you'll have to debug that on your own. If the `disable_csp` extension is missing, then either you modified your SeleniumBase installation (that extension comes with SeleniumBase) or you have a permissions issue that you must resolve own yourself.

A few other things:
* UC Mode doesn't support Edge.
* Mobile Mode with UC Mode could get you detected.
* Changing the User Agent in UC Mode could get you detected.

**Comment by drdedou** [comment]
how I get disable_csp  Because it does not download automatically 
downloaded_files/disable_csp

**Reply by mdmintz** [reply]
https://github.com/seleniumbase/SeleniumBase/blob/master/seleniumbase/extensions/disable_csp.zip
